app/src/main/java/com/yml/womensafety/AlertDialogUtil.kt

```diff
@@ -0,0 +1,28 @@
+package com.yml.womensafety
+
+
+import android.Manifest
+import android.app.Activity
+import android.content.Context
+import androidx.appcompat.app.AlertDialog
+import androidx.core.app.ActivityCompat
+
+object AlertDialogUtil {
+    private const val positiveMessage = "OK"
+    private const val LOCATION_PERMISSION_CODE = 1
+    fun showAlert(context: Context, title: CharSequence, message: CharSequence) {
```

Comment:
Good job. But try making it context extension function

Reply:
Done